packages/core-components/src/components/SelectElement/index.vue

```diff
@@ -231,8 +231,9 @@ const assignElements = (
 const isAllowedType = (el: ContentElement) => {
   if (!props.allowedElementConfig.length) return true;
   return props.allowedElementConfig.some((it: any) => {
-    const sameType = it.type === el.type;
-    const sameConfig = it.config.isGradable === el.data.isGradable;
+    const sameType = it.type === ceRegistry?.getByEntity(el).type;
```

Comment:
Should we move this logic to manifest (doesMatchAllowedElementConfig) in case rules change down the line and is reused to have check in a centralized place.